pilot/proxy/envoy/watcher.go

```diff
@@ -202,14 +202,29 @@ type envoy struct {
 	extraArgs []string
 }
 
+// ProxyLogLevel defines the value of the log level flag (-l) passed to the Envoy proxy upon creation.
+type ProxyLogLevel string
+
+const (
+	// ProxyLogLevelNone environment flag
```

Comment:
Maybe we should just pass a string? There is also notice, error, and warning levels, I think.

Reply:
done.